bridgebot/cardplay.py

```diff
@@ -111,4 +111,8 @@ def play(hands, contract, declarer):
 
         leading_player = winning_player
 
-    return {Team.NS: ns_tricks, Team.EW: ew_tricks}
+    if declarer in [Players.NORTH, Players.SOUTH]:
```

Comment:
We only need the declarer's tricks, since the defenders took 13 minus that number. It also makes the scoring functions cleaner.